msal/src/androidTest/java/com/microsoft/identity/client/DiscoveryTests.java

```diff
@@ -0,0 +1,27 @@
+package com.microsoft.identity.client;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import com.microsoft.identity.common.internal.providers.microsoft.azureactivedirectory.AzureActiveDirectory;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+
+@RunWith(AndroidJUnit4.class)
+public class DiscoveryTests {
+
+
+    @Test
+    public void testInstanceDiscovery() throws IOException {
```

Comment:
Additional tests we need to add here? Seems to only check that an Exception was not thrown

Reply:
Yes.  Let's discuss.